include/rocksdb/table.h

```diff
@@ -22,7 +22,7 @@
 #include <string>
 #include <unordered_map>
 
```

Comment:
Indicate this public API change in HISTORY.md

Reply:
It is already mentioned as part of the introduction of the Customizable class.  What else would you like?